website/docs/r/databricks_access_connector.html.markdown

```diff
@@ -51,25 +51,27 @@ The following arguments are supported:
 
 An `identity` block supports the following:
 
-* `type` - (Required) The type of identity to use for this Access Connector. `SystemAssigned` is the only possible value.
+* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on the Databricks Access Connector. Possible values include `SystemAssigned` or `UserAssigned`.
+
+* `identity_ids` - (Optional) Specifies a list of User Assigned Managed Identity IDs to be assigned to the Databricks Access Connector. Only one User Assigned Managed Identity ID is supported per Databricks Access Connector resource.
+
+~> **NOTE:** `identity_ids` are required when `type` is set to `UserAssigned`.
 
 ## Attributes Reference
 
-The following attributes are exported:
+The following Attributes are exported in addition to the Arguments listed above:
```

Comment:
(minor) looks like we use this phrasing in other resources, we should probably make this consistent:

```suggestion
In addition to the Arguments listed above - the following Attributes are exported:
```

Reply:
Fixed.